docs/products/clickhouse/get-started.md

```diff
@@ -5,77 +5,486 @@ sidebar_label: Get started
+
+```bash
+docker run -it                    \
+--rm clickhouse/clickhouse-client \
+--user=$CLICKHOUSE_USER           \
+--password=$CLICKHOUSE_PASSWORD   \
+--host=$CLICKHOUSE_HOST           \
+--port=$CLICKHOUSE_PORT           \
+--secure
+```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+You can verify your Aiven for ClickHouse connection from a Kubernetes workload by deploying
+a Pod that runs the `clickhouse-client` command.
+
+1. Create file `pod-clickhouse-client.yaml`:
+
+    ```yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: clickhouse-client-test-connection
+    spec:
+      restartPolicy: Never
+      containers:
+        - image: clickhouse-server:23.8
+          name: clickhouse
+          command: ["clickhouse-client", "$(DATABASE_URI)", "-c", "SELECT version();"]
+          envFrom:
+            - secretRef:
+                name: my-clickhouse-connection
     ```
 
-For more information on using the ClickHouse client, see
-[Connect to a ClickHouse® cluster with CLI](/docs/products/clickhouse/howto/connect-with-clickhouse-cli).
+   It runs once and stops due to the `restartPolicy: Never` flag.
+
+1. Inspect the log:
+
+   ```go
+   kubectl logs clickhouse-client-test-connection
+   ```
+
+   The output is similar to the following:
+
+   ```text
+                                              version
+   ---------------------------------------------------------------------------------------------
+   ClickHouse 23.8 on x86_64-pc-linux-gnu, compiled by gcc, a 68c5366192 p 6b9244f01a, 64-bit
+   (1 row)
+   ```
+
+You have now connected to your Aiven for ClickHouse service and executed
+the `SELECT version();` query.
+</TabItem>
+<TabItem value="4" label="ClickHouse client">
+[Connect to your new service with CLI](/docs/products/clickhouse/howto/connect-with-clickhouse-cli)
+using the
+[ClickHouse client](https://clickhouse.com/docs/en/integrations/sql-clients/cli).
+</TabItem>
+</Tabs>
+
+:::tip
+Discover more tools for connecting to Aiven for ClickHouse in
+[Connect to Aiven for ClickHouse®](/docs/products/clickhouse/howto/list-connect-to-service).
+:::
+
+## Load a dataset
+
+1. Download a dataset from
+   [Example Datasets](https://clickhouse.com/docs/en/getting-started/example-datasets/metrica/)
+   using cURL:
+
+   ```bash
+   curl address_to_file_in_format_tsv_xz | unxz --threads=`nproc` > file-name.tsv
+   ```
+
+   :::note
+   The `nproc` Linux command, which prints the number of processing units,
+   is not available on macOS. To use the command, add an alias for
+   `nproc` into your `~/.zshrc` file: `alias nproc="sysctl -n hw.logicalcpu"`.
+   :::
+
+   Once done, you should have two files: `hits_v1.tsv` and `visits_v1.tsv`.
+
+1. [Create database](/docs/products/clickhouse/howto/manage-databases-tables#create-a-clickhouse-database)
+   `datasets`.
+1. Create tables in the `datasets` database: `hits_v1` and `visits_v1`.
+
+   ```sql
+   CREATE TABLE datasets.hits_v1 [...]
```

Comment:
It is difficult to use this tutorial if the full queries are not here.

```sql
CREATE TABLE datasets.hits_v1 (
  WatchID UInt64,
  JavaEnable UInt8,
  Title String,
  GoodEvent Int16,
  EventTime DateTime,
  EventDate Date,
  CounterID UInt32,
  ClientIP UInt32,
  ClientIP6 FixedString(16),
  RegionID UInt32,
  UserID UInt64,
  CounterClass Int8,
  OS UInt8,
  UserAgent UInt8,
  URL String,
  Referer String,
  URLDomain String,
  RefererDomain String,
  Refresh UInt8,
  IsRobot UInt8,
  RefererCategories Array(UInt16),
  URLCategories Array(UInt16),
  URLRegions Array(UInt32),
  RefererRegions Array(UInt32),
  ResolutionWidth UInt16,
  ResolutionHeight UInt16,
  ResolutionDepth UInt8,
  FlashMajor UInt8,
  FlashMinor UInt8,
  FlashMinor2 String,
  NetMajor UInt8,
  NetMinor UInt8,
  UserAgentMajor UInt16,
  UserAgentMinor FixedString(2),
  CookieEnable UInt8,
  JavascriptEnable UInt8,
  IsMobile UInt8,
  MobilePhone UInt8,
  MobilePhoneModel String,
  Params String,
  IPNetworkID UInt32,
  TraficSourceID Int8,
  SearchEngineID UInt16,
  SearchPhrase String,
  AdvEngineID UInt8,
  IsArtifical UInt8,
  WindowClientWidth UInt16,
  WindowClientHeight UInt16,
  ClientTimeZone Int16,
  ClientEventTime DateTime,
  SilverlightVersion1 UInt8,
  SilverlightVersion2 UInt8,
  SilverlightVersion3 UInt32,
  SilverlightVersion4 UInt16,
  PageCharset String,
  CodeVersion UInt32,
  IsLink UInt8,
  IsDownload UInt8,
  IsNotBounce UInt8,
  FUniqID UInt64,
  HID UInt32,
  IsOldCounter UInt8,
  IsEvent UInt8,
  IsParameter UInt8,
  DontCountHits UInt8,
  WithHash UInt8,
  HitColor FixedString(1),
  UTCEventTime DateTime,
  Age UInt8,
  Sex UInt8,
  Income UInt8,
  Interests UInt16,
  Robotness UInt8,
  GeneralInterests Array(UInt16),
  RemoteIP UInt32,
  RemoteIP6 FixedString(16),
  WindowName Int32,
  OpenerName Int32,
  HistoryLength Int16,
  BrowserLanguage FixedString(2),
  BrowserCountry FixedString(2),
  SocialNetwork String,
  SocialAction String,
  HTTPError UInt16,
  SendTiming Int32,
  DNSTiming Int32,
  ConnectTiming Int32,
  ResponseStartTiming Int32,
  ResponseEndTiming Int32,
  FetchTiming Int32,
  RedirectTiming Int32,
  DOMInteractiveTiming Int32,
  DOMContentLoadedTiming Int32,
  DOMCompleteTiming Int32,
  LoadEventStartTiming Int32,
  LoadEventEndTiming Int32,
  NSToDOMContentLoadedTiming Int32,
  FirstPaintTiming Int32,
  RedirectCount Int8,
  SocialSourceNetworkID UInt8,
  SocialSourcePage String,
  ParamPrice Int64,
  ParamOrderID String,
  ParamCurrency FixedString(3),
  ParamCurrencyID UInt16,
  GoalsReached Array(UInt32),
  OpenstatServiceName String,
  OpenstatCampaignID String,
  OpenstatAdID String,
  OpenstatSourceID String,
  UTMSource String,
  UTMMedium String,
  UTMCampaign String,
  UTMContent String,
  UTMTerm String,
  FromTag String,
  HasGCLID UInt8,
  RefererHash UInt64,
  URLHash UInt64,
  CLID UInt32,
  YCLID UInt64,
  ShareService String,
  ShareURL String,
  ShareTitle String,
  ParsedParams Nested(
    Key1 String,
    Key2 String,
    Key3 String,
    Key4 String,
    Key5 String,
    ValueDouble Float64
  ),
  IslandID FixedString(16),
  RequestNum UInt32,
  RequestTry UInt8
)
ENGINE = MergeTree()
PARTITION BY toYYYYMM(EventDate)
ORDER BY
  (CounterID, EventDate, intHash32(UserID))
SAMPLE BY intHash32(UserID)
SETTINGS index_granularity = 8192;
```

Reply:
Thanks Joe! Could you please provide me with the parallel sample for the other CREATE TABLE statement?

Reply:
```sql
CREATE TABLE datasets.visits_v1 (
  CounterID UInt32,
  StartDate Date,
  Sign Int8,
  IsNew UInt8,
  VisitID UInt64,
  UserID UInt64,
  StartTime DateTime,
  Duration UInt32,
  UTCStartTime DateTime,
  PageViews Int32,
  Hits Int32,
  IsBounce UInt8,
  Referer String,
  StartURL String,
  RefererDomain String,
  StartURLDomain String,
  EndURL String,
  LinkURL String,
  IsDownload UInt8,
  TraficSourceID Int8,
  SearchEngineID UInt16,
  SearchPhrase String,
  AdvEngineID UInt8,
  PlaceID Int32,
  RefererCategories Array(UInt16),
  URLCategories Array(UInt16),
  URLRegions Array(UInt32),
  RefererRegions Array(UInt32),
  IsYandex UInt8,
  GoalReachesDepth Int32,
  GoalReachesURL Int32,
  GoalReachesAny Int32,
  SocialSourceNetworkID UInt8,
  SocialSourcePage String,
  MobilePhoneModel String,
  ClientEventTime DateTime,
  RegionID UInt32,
  ClientIP UInt32,
  ClientIP6 FixedString(16),
  RemoteIP UInt32,
  RemoteIP6 FixedString(16),
  IPNetworkID UInt32,
  SilverlightVersion3 UInt32,
  CodeVersion UInt32,
  ResolutionWidth UInt16,
  ResolutionHeight UInt16,
  UserAgentMajor UInt16,
  UserAgentMinor UInt16,
  WindowClientWidth UInt16,
  WindowClientHeight UInt16,
  SilverlightVersion2 UInt8,
  SilverlightVersion4 UInt16,
  FlashVersion3 UInt16,
  FlashVersion4 UInt16,
  ClientTimeZone Int16,
  OS UInt8,
  UserAgent UInt8,
  ResolutionDepth UInt8,
  FlashMajor UInt8,
  FlashMinor UInt8,
  NetMajor UInt8,
  NetMinor UInt8,
  MobilePhone UInt8,
  SilverlightVersion1 UInt8,
  Age UInt8,
  Sex UInt8,
  Income UInt8,
  JavaEnable UInt8,
  CookieEnable UInt8,
  JavascriptEnable UInt8,
  IsMobile UInt8,
  BrowserLanguage UInt16,
  BrowserCountry UInt16,
  Interests UInt16,
  Robotness UInt8,
  GeneralInterests Array(UInt16),
  Params Array(String),
  Goals Nested(
    ID UInt32,
    Serial UInt32,
    EventTime DateTime,
    Price Int64,
    OrderID String,
    CurrencyID UInt32
  ),
  WatchIDs Array(UInt64),
  ParamSumPrice Int64,
  ParamCurrency FixedString(3),
  ParamCurrencyID UInt16,
  ClickLogID UInt64,
  ClickEventID Int32,
  ClickGoodEvent Int32,
  ClickEventTime DateTime,
  ClickPriorityID Int32,
  ClickPhraseID Int32,
  ClickPageID Int32,
  ClickPlaceID Int32,
  ClickTypeID Int32,
  ClickResourceID Int32,
  ClickCost UInt32,
  ClickClientIP UInt32,
  ClickDomainID UInt32,
  ClickURL String,
  ClickAttempt UInt8,
  ClickOrderID UInt32,
  ClickBannerID UInt32,
  ClickMarketCategoryID UInt32,
  ClickMarketPP UInt32,
  ClickMarketCategoryName String,
  ClickMarketPPName String,
  ClickAWAPSCampaignName String,
  ClickPageName String,
  ClickTargetType UInt16,
  ClickTargetPhraseID UInt64,
  ClickContextType UInt8,
  ClickSelectType Int8,
  ClickOptions String,
  ClickGroupBannerID Int32,
  OpenstatServiceName String,
  OpenstatCampaignID String,
  OpenstatAdID String,
  OpenstatSourceID String,
  UTMSource String,
  UTMMedium String,
  UTMCampaign String,
  UTMContent String,
  UTMTerm String,
  FromTag String,
  HasGCLID UInt8,
  FirstVisit DateTime,
  PredLastVisit Date,
  LastVisit Date,
  TotalVisits UInt32,
  TraficSource Nested(
    ID Int8,
    SearchEngineID UInt16,
    AdvEngineID UInt8,
    PlaceID UInt16,
    SocialSourceNetworkID UInt8,
    Domain String,
    SearchPhrase String,
    SocialSourcePage String
  ),
  Attendance FixedString(16),
  CLID UInt32,
  YCLID UInt64,
  NormalizedRefererHash UInt64,
  SearchPhraseHash UInt64,
  RefererDomainHash UInt64,
  NormalizedStartURLHash UInt64,
  StartURLDomainHash UInt64,
  NormalizedEndURLHash UInt64,
  TopLevelDomain UInt64,
  URLScheme UInt64,
  OpenstatServiceNameHash UInt64,
  OpenstatCampaignIDHash UInt64,
  OpenstatAdIDHash UInt64,
  OpenstatSourceIDHash UInt64,
  UTMSourceHash UInt64,
  UTMMediumHash UInt64,
  UTMCampaignHash UInt64,
  UTMContentHash UInt64,
  UTMTermHash UInt64,
  FromHash UInt64,
  WebVisorEnabled UInt8,
  WebVisorActivity UInt32,
  ParsedParams Nested(
    Key1 String,
    Key2 String,
    Key3 String,
    Key4 String,
    Key5 String,
    ValueDouble Float64
  ),
  Market Nested(
    Type UInt8,
    GoalID UInt32,
    OrderID String,
    OrderPrice Int64,
    PP UInt32,
    DirectPlaceID UInt32,
    DirectOrderID UInt32,
    DirectBannerID UInt32,
    GoodID String,
    GoodName String,
    GoodQuantity Int32,
    GoodPrice Int64
  ),
  IslandID FixedString(16)
)
ENGINE = CollapsingMergeTree(Sign)
PARTITION BY toYYYYMM(StartDate)
ORDER BY
  (CounterID, StartDate, intHash32(UserID), VisitID)
```

Reply:
I just changed the original query too.